vstruct/defs/elf.py

```diff
@@ -80,6 +80,130 @@
     PT_HIPROC:"End of processor-specific",
 }
 
+DT_NULL     = 0
```

Comment:
why the duplication? these are defined in `Elf/elf_lookup.py`

Reply:
Same reason as the others. There's no good common place to put these, and I rather duplicate these than have a series of empty mixin classes. But I need them for around line 427.

Honestly, I'd prefer to move all of them to live under vstruct, but that might break some things we don't want to.